cmd/gke-gcloud-auth-plugin/gcloud_token_provider.go

```diff
@@ -25,11 +25,20 @@ type gcloudConfiguration struct {
 type gcloudTokenProvider struct {
 	readGcloudConfigRaw func() ([]byte, error)
 	readFile            func(filename string) ([]byte, error)
+	account             string
+	project             string
 }
 
 // readGcloudConfig returns an object which represents gcloud config output
 func (p *gcloudTokenProvider) readGcloudConfig() (*gcloudConfiguration, error) {
-	gcloudConfigBytes, err := p.readGcloudConfigRaw()
+	extraArgs := []string{}
+	if account != "" {
+		extraArgs = append(extraArgs, fmt.Sprintf("--account=%s", account))
+	}
+	if project != "" {
+		extraArgs = append(extraArgs, fmt.Sprintf("--project=%s", project))
+	}
+	gcloudConfigBytes, err := p.readGcloudConfigRaw(extraArgs)
```

Comment:
Is there a location where we can log the gcloud command being called? May be with klog.V(4).Infof("Gcloud command: ...... ")?

Reply:
I can add that in somewhere 👍